R/mainTests.r

```diff
@@ -1417,12 +1421,12 @@ region_combinations <- function(data) {
         filter(geographic_level == geography_matrix[2, 1]) %>%
         select(geography_matrix[2, 2], geography_matrix[2, 3]) %>%
         unique() %>%
-        filter(region_code != gssNAvcode && !is.na(region_code)),
+        filter(region_code != gssNAvcode | is.na(region_code)),
```

Comment:
just double check - was this an intended change (they look like they do different things but idk)

Reply:
Checked back and this was down to a change in behaviour of the code that reads in the data. It used to read blank entries as empty strings, but now reads them in as NAs. This bit of code was originally searching for the empty strings to identify blanks and this change allows it to still find the blanks by searching for NAs. There shouldn't be any valid reason for an NA in the region name or code fields if the geographic level is set to regional, so think this is safe. Good to have it described here now, so thanks for highlighting.